types/oidc-provider/index.d.ts

```diff
@@ -1213,7 +1213,7 @@ export type EncryptionEncValues =
 export interface InteractionResults {
     login?: {
         remember?: boolean;
-        account: string;
+        accountId: string;
```

Comment:
Sorry, I'm not familiar with this library, so this may be a dumb question.  The property you linked was in a `session`, whereas this appears to be in a `login` - is this the right property to update?  It looks like different parts of the library use either `account` or `accountId`, according to their needs.

Reply:
This change is correct.

Reply:
Thanks for confirming!